layers/+tools/lsp/packages.el

```diff
@@ -13,48 +13,29 @@
   '(
      lsp-mode
      lsp-ui
-     (lsp-imenu :requires imenu :location built-in)
-     (lsp-ui-imenu :requires lsp-imenu :location built-in)
      (company-lsp :requires company)
-     ;; `flycheck-lsp' does not exist so we defined it as built-in to avoid
-     ;; fetching it from ELPA repositories.
-     ;; this logical package serves to hook all flycheck related configuration
-     ;; for LSP.
-     (flycheck-lsp :requires flycheck :location built-in)
      ))
 
 (defun lsp/init-lsp-mode ()
   (use-package lsp-mode
     :defer t
     :config
     (progn
-      (spacemacs|hide-lighter lsp-mode)
+      (require 'lsp-clients)
+      (setq lsp-prefer-flymake nil)
+      (spacemacs/lsp-bind-keys)
+      (add-hook 'lsp-after-open-hook (lambda ()
+                                       "Setup xref jump handler and declare keybinding prefixes"
+                                       (spacemacs//setup-lsp-jump-handler major-mode)
+                                       (spacemacs//lsp-declare-prefixes-for-mode major-mode)))
       (evil-set-command-property 'lsp-goto-type-definition :jump t)
```

Comment:
`evil-set-command-property` has been added to lsp-mode.

Reply:
Thanks @MaskRay -- eliminated the duplication from packages.el